unstructured/partition/pdf.py

```diff
@@ -47,10 +68,30 @@ def partition_pdf(
         url=url,
         headers={"Authorization": f"Bearer {token}" if token else ""},
         files=file_,
+        data=data,  # NOTE(alan): Remove after unstructured API is using routing
     )
 
     if response.status_code == 200:
         pages = response.json()["pages"]
         return [element for page in pages for element in page["elements"]]
     else:
         raise ValueError(f"response status code = {response.status_code}")
+
+
+def _partition_pdf_via_local(
+    filename: str = "",
+    file: Optional[bytes] = None,
+    template: Optional[str] = None,
+) -> List[Element]:
+    """Partition using package installed locally."""
+    from unstructured_inference.inference.layout import (
```

Comment:
If this results in a `ModuleNotFoundError` or `ImportError` (I forget which one of those is more generally), should we raise an error with a message that tells the user to install the `local-inference` deps?

Reply:
Added error handling with instructions in [d5445d9](https://github.com/Unstructured-IO/unstructured/pull/125/commits/d5445d9661da22ad573229c57c3ef480544e94d4).